baseframe/forms/auto.py

```diff
@@ -18,24 +18,28 @@ class ConfirmDeleteForm(Form):
     cancel = SubmitField(__(u"Cancel"))
 
 
-def render_form(form, title, message='', formid=None, submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True, autosave=False):
+def render_form(form, title, message='', formid=None, submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True, action='', autosave=False, last_revision_id=None):
     multipart = False
     ref_id = 'form-' + (formid or buid())
+    if not action:
+        action = request.base_url
```

Comment:
On second thoughts, we should be using `request.url`, because query parameters may have significance as well. http://flask.pocoo.org/docs/1.0/api/#flask.Request.url